projects/WTI-API/src/main/controllers/ContestController.java

```diff
@@ -566,6 +573,30 @@ public Response getStandings(
 					// we got the internal contest; pass it to the DefaultScoringAlgorithm and get back updated standings
 					try {
 						Properties props = internalContest.getContestInformation().getScoringProperties();
+
+						xlog(logger, "debug 22 props "+props);
+						if (props != null) {
+						       Set<Object> set = props.keySet();
+
+						        String[] keys = (String[]) set.toArray(new String[set.size()]);
+
+						        Arrays.sort(keys);
+								xlog(logger, "debug 22   Scoring Properties, there are " + keys.length + " keys");
+
+						        for (String string : keys) {
+									xlog(logger, "debug 22      " + string + "='" + props.get(string) + "'");
+						        }
+						}
+						
+						props = DefaultScoringAlgorithm.getDefaultProperties();
```

Comment:
I do not understand why `props = DefaultScoringAlgorithm.getDefaultProperties` is _always_ being executed.  I believe it should only be executed IF the above call to `Properties props = internalContest.getContestInformation().getScoringProperties();` returns null.  Otherwise, the code is forcing the use of "default properties _even when valid properties have been obtained_.

Said another way, I think this line should be an `else` for the previous `if` statement, not an unconditionally executed statement.

Reply:
However, if what you are addressing is that the `ScoringProperties` object is coming back non-null, but one or more of the Properties _inside that object_ is null when it shouldn't be, then I guess this makes sense.

Reply:
Why unconditionally assignment?   Crunch time.  Needed to fix it to go onto the the next task.

Reply:
Also with a conditional there would be additional testing and one of the other fixes was to 
add a .ini entry (PR #682 ), and further there are three different integer scoring properties...  so
strictly it would be 3 conditionals and 3 tests.